modules/distribution/src/repository/resources/conf/deployment.toml

```diff
@@ -3,6 +3,10 @@ hostname = "localhost"
 node_ip = "127.0.0.1"
 base_path = "https://$ref{server.hostname}:${carbon.management.port}"
 
+[tenant_context]
```

Comment:
Let's enable it via the default.json

Reply:
Will enabled it via default.json with https://github.com/wso2/carbon-identity-framework/pull/5024